utils/create_dummy_models.py

```diff
@@ -410,7 +410,10 @@ def convert_processors(processors, tiny_config, output_folder, result):
         elif isinstance(processor, ProcessorMixin):
             # Currently, we only have these 2 possibilities
             tokenizers.append(processor.tokenizer)
-            feature_extractors.append(processor.feature_extractor)
+            if hasattr(processor, "image_processor"):
+                feature_extractors.append(processor.image_processor)
+            elif hasattr(processor, "feature_extractor"):
+                feature_extractors.append(processor.feature_extractor)
```

Comment:
the name of `feature_extractors` here is not good - it is a leftover when we didn't have image processor yet.